apps/web/components/booking/AvailableTimes.tsx

```diff
@@ -167,10 +167,10 @@ const AvailableTimes: FC<AvailableTimesProps> = ({
                         {!!seatsPerTimeSlot && (
                           <p
                             className={`${
-                              slot.attendees && slot.attendees / seatsPerTimeSlot >= 0.8
-                                ? "text-rose-600"
-                                : slot.attendees && slot.attendees / seatsPerTimeSlot >= 0.33
+                              slot.attendees && slot.attendees / seatsPerTimeSlot >= 0.5
                                 ? "text-yellow-500"
+                                : slot.attendees && slot.attendees / seatsPerTimeSlot >= 0.33
+                                ? "text-rose-600"
```

Comment:
Color fixing, was red at first and then yellow. I think it should be yellow and finally when running out of seats.